### PR TITLE
octopus: cephadm: Update images used

### PR DESCRIPTION
**Pull request by sebastian-philipp**
pseudo-backport of #34443 The content differs!

### CODE REVIEW

File: src/cephadm/cephadm [hunk 1]
```diff
@@ -1076,6 +1078,16 @@ def infer_fsid(func):
...
 
+def _get_default_image():
+    if DEFAULT_IMAGE_IS_MASTER:
+        yellow = '\033[93m'
+        end = '\033[0m'
+        s = '{}Warning: This is a development version of cephadm. Please refer to https://docs.ceph.com/docs/{}/cephadm/install{}'.format(
+            yellow, LATEST_STABLE_RELEASE, end
+        )
+        print(s)
+    return DEFAULT_IMAGE
```
Comment (mgfritch) [inline src/cephadm/cephadm:1089]: make it fancy? https://github.com/ceph/ceph/pull/34693

**Comment by sebastian-philipp** [comment]
QA run: http://pulpito.ceph.com/swagner-2020-04-23_10:07:35-rados-wip-swagner2-testing-2020-04-22-1515-distro-basic-smithi/

Fixes: http://pulpito.ceph.com/teuthology-2020-04-20_03:30:03-rados-octopus-distro-basic-smithi/4968464/ (cc @neha-ojha )

Failures:
* https://tracker.ceph.com/issues/45194
* https://tracker.ceph.com/issues/44715

**Comment by smithfarm** [comment]
> pseudo-backport of #34443 The content differs!

This is supposed to be reflected in the commit message, actually - by including a "Conflicts:" section.

```
* the commit message generated by git cherry-pick -x must not be modified, except to add a "Conflicts" section below the "cherry picked from commit ..." line added by git
* the "Conflicts" section must mention all files where changes had to be made manually (not just conflicts flagged by git)
* the "Conflicts" section should also describe the manual changes that were made
```

**Comment by sebastian-philipp** [comment]
> > pseudo-backport of #34443 The content differs!
> 
> This is supposed to be reflected in the commit message, actually - by including a "Conflicts:" section.
> 
> ```
> * the commit message generated by git cherry-pick -x must not be modified, except to add a "Conflicts" section below the "cherry picked from commit ..." line added by git
> * the "Conflicts" section must mention all files where changes had to be made manually (not just conflicts flagged by git)
> * the "Conflicts" section should also describe the manual changes that were made
> ```

you're right.

**Pull request merged**